pkg/runner/cluster_k8s.go

```diff
@@ -141,8 +142,15 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 
 	cfg := *input.RunnerConfig.(*ClusterK8sRunnerConfig)
 
-	defaultCPU := resource.MustParse(cfg.PodResourceCPU)
-	defaultMemory := resource.MustParse(cfg.PodResourceMemory)
+	var defaultCPU, defaultMemory resource.Quantity
```

Comment:
There are no defautls for defaultCPU and defaultMemory and without having them set in .env.toml causes an error, so I am doing this to create defaults.

the local:docker runner handles this differently. It has a default config struct and uses `mergo` to merge the defaults with supplied config. maybe this should be made the same.

Reply:
Yes, this is an oversight on my part. We should have defaults, and ideally the same way as other runners.

Reply:
The only reason I left this like that, is because in general when you run `cluster:k8s`, so far I assumed it'd be AWS (as we don't have another setup until now), and you also had to specify `aws region` - therefore the `.env.toml` is really required when using `cluster:k8s`.